harness/determined/common/experimental/determined.py

```diff
@@ -130,8 +133,13 @@ def create_experiment(
             config(string, pathlib.Path, dictionary): experiment config filename (.yaml)
                 or a dict.
             model_dir(string, optional): directory containing model definition. (default: ``None``)
-            includes (Iterable[Union[str, pathlib.Path]], optional): Additional files or
-            directories to include in the model definition.  (default: ``None``)
+            includes(Iterable[Union[str, pathlib.Path]], optional): Additional files or
+            directories to include in the model definition. (default: ``None``)
+            parent_id(int, optional): The id of the parent experiment. (default: ``None``)
+            project_id(int, optional): The id of the project this experiment should belong to.
+            (default: ``None``)
+            template(string, optional): The name of the tamplate for the experiment.
```

Comment:
nit: template spelling

In the context of someone coming to [the SDK reference](https://docs.determined.ai/latest/reference/python-sdk/python-sdk.html), I don't think templates are easily understood. Since we're giving this kind of template more visibility than it had, could you please work on the user experience a little bit?

This could mean finding a way to reference https://docs.determined.ai/latest/model-dev-guide/dtrain/config-templates.html in this docstring, or it might just mean referring to them as "configuration templates" in the description rather than just template. I honestly don't know -- I don't understand templates very well. Tara might have opinions.

I think it'd also be useful in the docstring to describe what happens when the optional arguments aren't passed. At least in those cases where it isn't obvious to the user.

Reply:
I added a reference as previewed below
<img width="822" alt="Screenshot 2024-03-12 at 2 43 47 PM" src="https://github.com/determined-ai/determined/assets/40620519/1e10146b-eb3f-4bc7-9b4d-9db2f2abadff">

Reply:
Thanks! I think this looks great!